src/expressions/operator-expr.md

```diff
@@ -70,6 +70,50 @@ let a = && && mut 10;
 let a = & & & & mut 10;
 ```
 
+### Raw address-of operators
+
+Related to the borrow operators are the *raw address-of operators*, which do not have first-class syntax, but are exposed via the macros `ptr::addr_of!(expr)` and `ptr::addr_of_mut!(expr)`.
+Like with `&`/`&mut`, the expression `expr` is evaluated in place expression context.
```

Comment:
The first and third sentences together seem to make up the main definition and then this second sentence is just awkwardly in the middle. Perhaps the second and third sentences should be swapped.

Reply:
I don't know what you mean -- the operand, `expr`, is named, isn't it?

Reply:
I made the initial definition self-contained and moved the references to the borrow operator to the second paragraph.